app/src/main/java/com/oybekdev/plant_app/presentation/sign_up/SignUpScreen.kt

```diff
@@ -0,0 +1,95 @@
+package com.oybekdev.plant_app.presentation.sign_up
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.rememberNavController
+import com.oybekdev.plant_app.presentation.sign_up.component.SignUpButton
+import com.oybekdev.plant_app.presentation.sign_up.component.SingUpTextField
+
+@Composable
+fun SignUpScreen(
+    navController: NavController
```

Comment:
How about to add `modifier` to SignUpScreen?

`fun SignUpScreen(
 modifier: Modifier = Modifier
)
`